.github/workflows/release.yaml

```diff
@@ -113,46 +113,35 @@ jobs:
     needs: goreleaser
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout WG
-        uses: actions/checkout@v2
-        with:
-          path: main-repo
-      - name: Checkout Docs
+      - name: Checkout
         uses: actions/checkout@v2
-        with:
-          repository: weaveworks/weave-gitops-docs
-          path: doc-repo
-          token: ${{ secrets.WEAVE_GITOPS_DOCS_WEAVEWORKS_DOCS_BOT_PAT }}
       - name: Unshallow
         run: |
-          cd main-repo
           git fetch --prune --unshallow
       - name: Setup Go
         uses: actions/setup-go@v2
         with:
           go-version: 1.17.x
       - name: Create GitOps binary
         run: |
-          cd main-repo
           make all
       - name: Set up environment vars
         run: |
           echo "BRANCH=release-doc-updates-$$" >> $GITHUB_ENV
-          cd main-repo
           echo "GITOPS_VERSION=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
       - name: Update with new version
         env:
           ALGOLIA_API_KEY: ${{ secrets.ALGOLIA_API_KEY }}
+          GA_KEY: ${{ secrets.GA_KEY }}
         run: |
-          bash main-repo/tools/update-docs.sh ${PWD}/main-repo/bin/gitops ${PWD}/doc-repo
+          bash tools/update-docs.sh ${PWD}/bin/gitops ${PWD}/website
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@9825ae65b1cb54b543b938503728b432a0176d29 # v3
         with:
-          token: ${{ secrets.WEAVE_GITOPS_DOCS_WEAVEWORKS_DOCS_BOT_PAT }}
+          token: ${{ secrets.WEAVEWORKS_DOCS_PR_TOKEN }}
```

Comment:
Can we just keep using `WEAVE_GITOPS_DOCS_WEAVEWORKS_DOCS_BOT_PAT` here?

The naming is a bit untidy but the bot might/should be able to create PRs on weave-gitops as its a member of weaveworks? 🤔

Reply:
good call, I think we can, since that's listed as an organization token. I'll update it.